src/databricks/labs/ucx/source_code/linters/ast_helpers.py

```diff
@@ -1,30 +1,28 @@
-import ast
+from astroid import Attribute, Call, Name  # type: ignore
 
 
 class AstHelper:
     @staticmethod
-    def get_full_attribute_name(node: ast.Attribute) -> str:
-        return AstHelper._get_value(node)
+    def get_full_attribute_name(node: Attribute) -> str:
+        return AstHelper._get_attribute_value(node)
```

Comment:
if you turn this into `@classmethod`, then you can use `cls` ref instead of explicit `AstHelper` (which looks odd in the body of the same class).

Reply:
Fixed